File: Grade 11/AS work/Secret number.py
Pass = 0
SecretNumber = 56982
Guess = 0
while Pass == 0:
	Guess = input ("Please enter guess:")
	if int(Guess) > int(SecretNumber):
		print ("Secret number is smaller")
		Pass = 0
	elif int(Guess) < int(SecretNumber):
		print ("Secret number is greater")
		Pass = 0
	else:
		print ("You've guessed the secret number!")
		Pass = 1

File: Grade 11/AS work/Secret number.vb
Module Module1

    Sub Main()
        Dim Pass As Integer
        Dim SecretNumber As Integer
        Dim Guess As Integer
        Pass = 0
        SecretNumber = 56982
        While Pass = 0
            Console.WriteLine("Please enter guess:")
            Guess = Console.ReadLine
            If Guess > SecretNumber Then
                Console.WriteLine("Secret number is smaller")
            ElseIf Guess < SecretNumber Then
                Console.WriteLine("Secret number is greater")
            Else
                Console.WriteLine("You've guessed the secret number!")
                Pass = 1
            End If
        End While

    End Sub

End Module
